awx/main/migrations/_create_system_jobs.py

```diff
@@ -15,9 +16,41 @@
 only if new system job templates were created (i.e. new database).
 '''
 
+def create_collection_jt(apps, schema_editor):	
+    SystemJobTemplate = apps.get_model('main', 'SystemJobTemplate')	
+    Schedule = apps.get_model('main', 'Schedule')	
+    ContentType = apps.get_model('contenttypes', 'ContentType')	
+    sjt_ct = ContentType.objects.get_for_model(SystemJobTemplate)	
+    now_dt = now()	
+    random_time = now_dt + timedelta(minutes=random.randint(-30,30))	
+    random_schedule_time = random_time.strftime('%Y%m%dT%H%M%SZ')
+    insights_enabled = getattr(settings, 'INSIGHTS_TRACKING_STATE', False)
 
-def create_clearsessions_jt(apps, schema_editor):
+    sjt, created = SystemJobTemplate.objects.get_or_create(	
+        job_type='gather_analytics --ship',
+        defaults=dict(
+            name='Automation Insights Collection',
+            description='Collect analytics data and send it to Automation Insights',	
+            polymorphic_ctype=sjt_ct,
+            created=now_dt,
```

Comment:
Are `created` and `modified` necessary to pass in, or does the ORM just handle this for us on a `SystemJobTemplate`?